include/tscore/Extendible.h

```diff
@@ -141,9 +148,11 @@ namespace details // internal stuff
   {
   public:
     std::unordered_map<std::string, FieldDesc> fields; ///< defined elements of the blob by name
-    size_t alloc_size               = 0;               ///< bytes to allocate for fields
-    uint8_t alloc_align             = 1;               ///< alignment of block
-    std::atomic_uint instance_count = {0};             ///< the number of Extendible<Derived> instances in use.
+    size_t alloc_size        = 0;                      ///< bytes to allocate for fields
+    uint8_t alloc_align      = 1;                      ///< alignment of block
+    uint cnt_constructed     = {0};                    ///< the number of Extendible<Derived> created.
+    uint cnt_fld_constructed = {0};                    ///< the number of Extendible<Derived> that constructed fields.
```

Comment:
Shouldn't these be `atomic`?

Reply:
do they need to be atomic if they only increment?